### ISSUE TITLE
Chords involving left and right arrows are reported incorrectly

### ISSUE THREAD

**Issue by aarzilli** [skelterjohn/go.wde#30]
When pressing ctrl+right arrow I get control+arrow instead of control+right_arrow. It's a bug in ConstructChord:

```
 --- a/keys.go
+++ b/keys.go
@@ -164,10 +164,10 @@ func (c ChordSorter) Less(i, j int) (less bool) {
 func ConstructChord(keys map[string]bool) (chord string) {
        unikeys := map[string]bool{}
        for key := range keys {
-               if strings.HasPrefix(key, "left_") {
+               if strings.HasPrefix(key, "left_") && (key != "left_arrow"){
                        key = key[5:]
                }
-               if strings.HasPrefix(key, "right_") {
+               if strings.HasPrefix(key, "right_") && (key != "right_arrow") {
                        key = key[6:]
                }
                unikeys[key] = true
```

**Comment by sqweek** [comment]
Aha! I noticed the same thing but went with the opposite fix (only trim the prefix if the key is a "modifier"). skelterjohn is busy with other projects these days so I've published the changes on my fork.

**Comment by sqweek** [comment]
Fixed by b2d8664.
